CMakeLists.txt

```
@@ -0,0 +1,52 @@
+cmake_minimum_required (VERSION 3.15)
```

is `3.15` really required? It's really a pain to upgrade on ubuntu

```suggestion
cmake_minimum_required (VERSION 3.10)
```